src/storybook/stand-alone-documentaion/accessibility/accessibility.stories.mdx

```diff
@@ -1,13 +1,165 @@
+* Should use ``role="presentation"``
+
+<Frame className="monday-storybook-frame--no-border monday-storybook-frame--no-gutter">
+    <img src={imageryDecorative} alt="example decorative image" role="presentation"/>
+</Frame>
+
+## Hierarchy and focus
+__Reading Order__<br/>
+The reading and navigation order is determined by code order<br/>
+The reading order must be logical and intuitive.<br/>
+We arrange the order of the elements within the HTML so that the reading order follows the same order as the visual presentation of the screen.
+
+<Frame className="monday-storybook-frame--no-border monday-storybook-frame--no-gutter">
+    <img src={hierarchyOrder} alt="example decorative image" role="presentation"/>
+</Frame>
+
+__Semantic Markup__<br/>
+Semantic markup is used to define: Headings, Regions/Landmarks, Lists, Emphasized or special text, Tables and
+Text labels & form group labels
+
+__Headings__<br/>
+Headings communicate the structure of the content on the page. Assistive technologies users can use them as in-page navigation elements.
+* Arrange the headings on the page according to their level (``h1...h6``).
+* The most important heading on the page is heading level 1 (``<h1>``).
+* Only one heading level 1 ``<h1>`` is allowed on a page.
+* Headings should be placed at the start of new sections of content not for decoration.
+* Avoid skipping heading levels. Ensure that a heading level 1 ``<h1>`` is not followed directly by an ``<h3>``, for example.
+
+## Navigation
+__Keyboard Access__<br/>
+All active elements that can be activated with a mouse have to be operable by keyboard, or have a keyboard equivalent.<br/>
+Dynamic components such as dialogs, not initially in the tab order must also receive keyboard focus.<br/>
+Elements that show content on mouse hover needs to be added to the tab-order and be operable with the keyboard.<br/>
+In your design, annotate the tab-order and number the tab stops.
+
+<Frame className="monday-storybook-frame--no-border monday-storybook-frame--no-gutter">
+    <img src={navigationTabs} alt="example for keyboard navigation" role="presentation"/>
+</Frame>
+
+__Focus Management__<br/>
+Keyboard navigation in a rich internet application is different from tabbing in a website.<br/>
+In rich internet applications, users tab to complex components such as comboboxes, grids, menus etc. then use the arrow keys to navigate within the component.<br/>
+Dynamic components such as dialogs, not initially in the tab order have to receive keyboard focus.<br/>
+When dynamic components are closed, focus shifts back to the point where interaction started.<br/>
+Receiving focus or interacting with an input element must not result in a substantial change to the page like invoking a dialog or sending focus somewhere else on the page.
+
+<Tip title="We got you covered with tab navigation">Use this keyboard navigation infra in any UI that requires it.</Tip>
+
+__Focus Visible__<br/>
```

Comment:
since this all are headings and we in the a11y page, why not to use heading semantic tags h1...h5 ?
and add the css as class (maybe create sub header component)